build/tasks/clang-format.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+var gulp = require('gulp');
```

Comment:
@jeeyun: Quick question. Do we not need a clang format config file for setting things that we want to check?